application/aam-backend-service/src/main/kotlin/com/aamdigital/aambackendservice/reporting/reportcalculation/core/DefaultReportCalculator.kt

```diff
@@ -55,18 +56,26 @@ class DefaultReportCalculator(
 
     private fun setToDateToLastMinuteOfDay(args: MutableMap<String, String>) {
         val toDateString = args["to"] ?: return
-        args["to"] = toDateString.substring(IntRange(0, 9)) + "T23:59:59.999Z"
+        args["to"] = toDateString.substring(0, INDEX_ISO_STRING_DATE_END) + "T23:59:59.999Z"
     }
 
-    private fun getReportCalculationArgs(neededArgs: List<String>, givenArgs: Map<String, String>): List<String> =
-        neededArgs
+    private fun getReportCalculationArgs(
```

Comment:
shall we document this in a method signature comment and the OpenApi specs, what the exact behavior of the from-to currently is?